src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/ProjectSteps.java

```diff
@@ -14,13 +19,194 @@
 
     @Autowired
     private Project project;
+
+    @Autowired
+    private Header header;
+
+    @Autowired
+    private ConfirmAction confirm;
+    static final String PROJECTURI = "/projects";
+
     /**
      * Create a new project.
      *
      * @param projectAttributes for create project.
      */
-    @Given("user creates a default project as")
+    @When("user creates project as")
     public void userCreatesNewProjectAs(final Map<String, String> projectAttributes) {
         project.createNewProject(projectAttributes);
     }
+
+    /**
+     * was project created with specific name.
+     */
+    @Then("validate creation on project's dashboard")
+    public void validateTheProjectIsCreatedWithSpecifyName() {
+        final String actual = project.getProjectNameOnDashboard();
+        Assert.assertEquals(actual, project.getProjectName(), "Project name match");
+    }
+
+    /**
+     * A project to delete.
+     *
+     * @param projectName the project name
+     */
+    @Given("an existing project named as {string} that user intends to delete")
```

Comment:
I don't understand this precondition, where are you creating the project to be deleted? 
please create your precondition data with post step

Reply:
and remove it